webserver/cWebem.cpp

```diff
@@ -770,7 +770,7 @@ namespace http {
 				std::string attachment;
 				for (const auto &header : rep.headers)
 				{
-					if (boost::iequals(header.name, "Content-Disposition"))
+					if (header.name == "Content-Disposition")
```

Comment:
This wont work as it is a case insensitive compare...
You could add the below two functions to helper.h/cpp and use those:

bool iequals(const std::string &Input, const std::string &Test)
{
	return Input.size() == Test.size()
	       && equal(Input.cbegin(), Input.cend(), Test.cbegin(),
			[](std::string::value_type l1, std::string::value_type r1) {
		       return toupper(l1) == toupper(r1);
	       });
}

bool iequals(const std::wstring &Input, const std::wstring &Test)
{
	return Input.size() == Test.size()
	       && equal(Input.cbegin(), Input.cend(), Test.cbegin(),
			[](std::wstring::value_type l1, std::wstring::value_type r1) { return towupper(l1) == towupper(r1); });
}

Reply:
This will take a while. I need to rebuild my dev machine.

Reply:
I got rid of this change. Maybe will tackle at another time...